spec/views/sign_up/passwords/new.html.slim_spec.rb

```diff
@@ -4,7 +4,7 @@
   before do
     user = build_stubbed(:user)
     allow(view).to receive(:current_user).and_return(nil)
-    allow(view).to receive(:session).and_return(sign_up_init: true)
+    allow(view).to receive(:params).and_return(confirmation_token: 123)
```

Comment:
Nope! The modal is still on this screen. We are just making use of the existing `confirmation_token` session key, rather than `sign_up_init`, which isn't needed anymore.